samples/dnn/siamrpnpp.py

```diff
@@ -325,19 +323,19 @@ def main():
     """
     parser = argparse.ArgumentParser(description='Use this script to run SiamRPN++ Visual Tracker',
                                      formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-    parser.add_argument('--input_video', type=str, required=True, help='Path to input video file. Skip this argument to capture frames from a camera.')
+    parser.add_argument('--input_video', type=str, help='Path to input video file. Skip this argument to capture frames from a camera.')
     parser.add_argument('--target_net', type=str, default='target_net.onnx', help='Path to part of SiamRPN++ ran on target frame.')
     parser.add_argument('--search_net', type=str, default='search_net.onnx', help='Path to part of SiamRPN++ ran on search frame.')
     parser.add_argument('--rpn_head', type=str, default='rpn_head.onnx', help='Path to RPN Head ONNX model.')
     args, _ = parser.parse_known_args()
 
-    if not os.path.isfile(args.input_video):
+    if args.input_video and not os.path.isfile(args.input_video):
         raise OSError("Input video file does not exist")
-    if not os.path.isfile(args.target_net):
+    if args.target_net not os.path.isfile(args.target_net):
```

Comment:
`args.target_net` has default value `target_net.onnx`.  Thus, we can only check  `os.path.isfile(args.input_video)`